orca-sql/src/main/resources/db/changelog/20190911-notification-cluster-lock.yml

```diff
@@ -14,6 +14,6 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: expiry
-                  type: bigint(13)
+                  type: bigint
```

Comment:
Won't `validCheckSum` be required here as well?

Reply:
Indeed it would be - I've added the original checksum to the migration.